expression/explain.go

```diff
@@ -65,6 +65,51 @@ func (expr *ScalarFunction) ExplainNormalizedInfo() string {
 	return expr.explainInfo(true)
 }
 
+// ExplainInfo implements the Expression interface for plan digest.
+func (expr *ScalarFunction) ExplainInfo4InList() string {
+	return expr.explainInfo4InList(false)
+}
+
+func (expr *ScalarFunction) explainInfo4InList(normalized bool) string {
+	var buffer bytes.Buffer
+	fmt.Fprintf(&buffer, "%s(", expr.FuncName.L)
+	switch expr.FuncName.L {
+	case ast.Cast:
+		for _, arg := range expr.GetArgs() {
+			if normalized {
+				buffer.WriteString(arg.ExplainNormalizedInfo())
+			} else {
+				buffer.WriteString(arg.ExplainInfo())
+			}
+			buffer.WriteString(", ")
+			buffer.WriteString(expr.RetType.String())
+		}
+	default:
+		if expr.FuncName.L == ast.In {
+			buffer.WriteString("...")
```

Comment:
What if there is only one constant or if there are columns in the arguments? What's the expected behavior?

Reply:
I think we can say this is proper and expected behavior. And I'm also OK with this rationale.
What I'm concerned about is that it's inconsistent with the behavior of SQL digest. Essentially there isn't an overall design for their behaviors.